packages/terra-clinical-application-site/src/index.scss

```diff
@@ -16,6 +16,7 @@ body {
   height: 100%;
 }
 
-.terraClinical-ItemView-accessory {
+[class*="ItemView__accessory"],
+[class*="ItemView__accessory"] > * {
```

Comment:
This seems a little out of place, what is this doing exactly?

This wildcard class selector is dependent on the css-loader options defined in the webpack config. If the option changes this css would stop working.